cypress/support/ethereum.ts

```diff
@@ -16,7 +16,7 @@ const TEST_PRIVATE_KEY = '0xe580410d7c37d26c6ad1a837bbae46bc27f9066a466fb3a66e77
 // address of the above key
 const TEST_ADDRESS_NEVER_USE = new Wallet(TEST_PRIVATE_KEY).address
 const CHAIN_ID = SupportedChainId.GOERLI
-const HEXLIFIED_CHAIN_ID = `$0x${CHAIN_ID}`
+const HEXLIFIED_CHAIN_ID = `0x${CHAIN_ID}`
```

Comment:
For values greater than 9, this will ensure it remains hexlified:

```suggestion
const HEXLIFIED_CHAIN_ID = `0x${CHAIN_ID.toString(16)}`
```